sbt-init.sh

```diff
@@ -3,8 +3,8 @@
 source $SDKMAN_DIR/bin/sdkman-init.sh
 
 # Install Java and SBT - Reference: https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html
-# By default, it installs the latest version of Java 8 from OpenJDK (jdk.java.net)
-sdk install java 8.0.342-amzn
+# By default, it installs the latest version of Java 17 from OpenJDK (jdk.java.net)
+sdk install java 17.0.4-amzn
 sdk install sbt 1.7.1
```

Comment:
We should update this file to load these values from the parameters instead of having them hardcoded.

Reply:
Agreed. I tried to do initially but then realized that it was a larger change than I thought it would be.